src/datasets/table.py

```diff
@@ -1916,7 +1916,7 @@ def _concat_arrays(arrays):
                 _concat_arrays([array.values for array in arrays]),
             )
         elif pa.types.is_fixed_size_list(array_type):
-            if config.PYARROW_VERSION.major < 13:
+            if config.PYARROW_VERSION.major < 14:
```

Comment:
IMO this bug is too specific to run such a check.

Reply:
It would be quite simple to check though no ? And it can be run only once the first time we need to check if the bug exists.

Anyway let's merge this PR ^^